setup/install/storage/oracle.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: single
+title:  "Oracle Cloud Infrastructure Object Storage"
+sidebar:
+  nav: setup
+---
+
+Using [Oracle Object Storage](https://docs.cloud.oracle.com/iaas/Content/Object/Concepts/objectstorageoverview.htm){:target="\_blank"} as a storage source means that Spinnaker will store all of its persistent data in a
+[Bucket](https://docs.cloud.oracle.com/iaas/Content/Object/Tasks/managingbuckets.htm){:target="\_blank"}.
+
+## Prerequisites
+
+If you have enabled [Oracle Cloud provider](/setup/install/providers/oracle/) in Spinnaker, you may use the same region, Tenancy’s OCID, user’s OCID, Compartment's OCID, private key file, and fingerprint to enable Oracle Object Storage. You will need the following to enable Oracle Object Storage in Spinnaker:
```

Comment:
For consistency, suggest using ASCII apostrophe (') for all apostrophes and letting the front-end make it a curly apostrophe.